pkg/graveler/ref/manager.go

```diff
@@ -15,22 +15,25 @@ import (
 	"github.com/treeverse/lakefs/pkg/graveler"
 	"github.com/treeverse/lakefs/pkg/ident"
 	"github.com/treeverse/lakefs/pkg/kv"
+	"github.com/treeverse/lakefs/pkg/logging"
 )
 
-// MaxBatchDelay - 3ms was chosen as a max delay time for critical path queries.
-// It trades off amount of queries per second (and thus effectiveness of the batching mechanism) with added latency.
-// Since reducing # of expensive operations is only beneficial when there are a lot of concurrent requests,
-//
-//	the sweet spot is probably between 1-5 milliseconds (representing 200-1000 requests/second to the data store).
-//
-// 3ms of delay with ~300 requests/second per resource sounds like a reasonable tradeoff.
-const MaxBatchDelay = time.Millisecond * 3
-
-// commitIDStringLength string representation length of commit ID - based on hex representation of sha256
-const commitIDStringLength = 64
-
-// LinkAddressTime the time address is valid from get to link
-const LinkAddressTime = 6 * time.Hour
+const (
+	// MaxBatchDelay - 3ms was chosen as a max delay time for critical path queries.
+	// It trades off amount of queries per second (and thus effectiveness of the batching mechanism) with added latency.
+	// Since reducing # of expensive operations is only beneficial when there are a lot of concurrent requests,
+	//
+	//	the sweet spot is probably between 1-5 milliseconds (representing 200-1000 requests/second to the data store).
+	//
+	// 3ms of delay with ~300 requests/second per resource sounds like a reasonable tradeoff.
+	MaxBatchDelay = time.Millisecond * 3
```

Comment:
Suggest for readability
```suggestion
	MaxBatchDelay = 3 * time.Millisecond
```